config/Version.public.xcconfig

```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=16.0
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=16.0.0.0
+VERSION_LONG=16.0.0.2
```

Comment:
I was surprised by this increment, but then I saw that there's a commit that brings it to `16.0.0.1` first 👍